mantidproject/mantid
# Avoid linking to both libomp and libgomp on Ubuntu 18.04

**Description of work.**

Removes our custom FindGSL module. See commit description for reasoning.

**Report to:** [nobody]. 

**To test:**

A clean build (including removal of the cmake tree first) of at least `KernelTest` would be a good idea. On Ubuntu 18.04 the following command

```
ldd bin/KernelTest | grep omp
```

should now only produce a single entry containing `libgomp`. 

*There is no associated issue.*

*This does not require release notes* because **it is an internal change**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
